src/vm/arm/stubs.cpp

```diff
@@ -534,13 +534,21 @@ void LazyMachState::unwindLazyState(LazyMachState* baseState,
 
     do
     {
+#ifdef DACCESS_COMPILE
```

Comment:
Actually, looking at the implementation of Thread::VirtualUnwindCallFrame, we can call it for FEATURE_DACCESS as well. So all you need to do is to change the `#ifndef FEATURE_PAL` to `#if !defined(FEATURE_PAL) || defined(DACCESS_COMPILE)`